mobbex-for-woocommerce.php

```diff
@@ -258,10 +258,11 @@ public static function notice($type, $msg)
      * @access public
      */
     function additional_button_add_to_cart() {
+        
         ?>
         <Style>
             /* The Modal (background) */
-            .modal {
```

Comment:
Para empezar a achicar un poco el archivo principal, estaría bueno si podemos ir moviendo los estilos y el html a archivos distintos en una carpeta assets. Otra recomendación sería añadir el la condición directamente al add_filter (en la línea 59), de forma de que no se ejecute directamente la lógica del hook.
Un ejemplo de como podría quedar:
`$helper = new MobbexHelper();`
`if ($helper->finnancial_info && $helper->tax_id) {`
`  add_action( 'woocommerce_after_add_to_cart_form', [$this,'additional_button_add_to_cart'], 20 );`
`}`
Para esto habría que agregar las configs que se necesitan al helper también.

Reply:
¿Agregar eso,  hace que no se instale el hook en la instalación?, ósea el init() se ejecuta solo en la instalación

Reply:
El método init se ejecuta cada vez que se instancia el plugin. Wordpress entra al archivo y lo lee hasta el final, donde se añade un hook para que inicie la clase MobbexGateway con ese método